examples/source/1.components/amp-bind-recaptcha.html

```diff
@@ -0,0 +1,113 @@
+<!---
+
+author: elisameyer
+
+--->
+<!--
+  ## Introduction
+
+  This example shows how to create recaptcha input using just amp-bind.
+-->
```

Comment:
add an empty comment afterwards to avoid that the doctype will be displayed:

```
<!-- -->
```

Reply:
updated